js/bytex.js

```diff
@@ -1028,14 +1124,14 @@ module.exports = class bytex extends Exchange {
     }
 
     async cancelAllOrders (symbol = undefined, params = {}) {
-        await this.loadMarkets ();
         if (symbol === undefined) {
-            throw new ArgumentsRequired (this.id + ' cancelAllOrders() requires a symbol argument');
+            throw new ArgumentsRequired (this.id + " cancelAllOrders() requires a 'symbol' argument");
         }
-        const market = this.market (symbol);
-        const request = {
-            'symbol': market['id'],
-        };
```

Comment:
Symbol is required, so this is the correct way to do this, there's no need to set market to undefined first, and this is how we do this on all other exchanges